### DISCUSSION TITLE
How to

### DISCUSSION THREAD

**Discussion by onlyv8** [frtz13/EnergyRouter#1] [Ideas]
Bonjour frtz13,

Ce petit message pour vous féliciter pour le travail accompli !!!

Cela fait 4 mois que je cherche de quoi gérer mon surplus solaire avec HA, esphome sur esp82636, et un robotdyn.... Alleluia, je l'ai trouvé !!!

Seulement, je ne sais pas par où commencer, je suis un newbie sur HA, et là, très sincèrement, sans vous offenser, je ne vois pas...

Très sincèrement, votre projet mérite d'être connu et reconnu, car il utilise des éléments techniques simples que beaucoup de gens peuvent facilement avoir, et ce, sans dépenser vraiment beaucoup.

En attendant de pouvoir installer votre projet, je module moi -même à distance mon ballon, mais franchement, ce n'est pas top.

Bravo encore pour ce projet !!!

Eric

**Comment by frtz13** [comment]
merci, Eric.
... par où commencer... ?
- installer Mosquitto add-on
- installer le Shelly-EM, le connecter au broker MQTT (Mosquitto), et mettre en place les Sensors MQTT dans H.A. / se familiariser avec MQTT-Explorer
- construire le dimmer, et le programmer avec ESPHome
- Faire tourner le routeur d'énergie dans AppDaemon, ou autrement

**Comment by onlyv8** [comment]
Bonjour FRTZ13,

Je suis désolé, mais je n'arrive pas à comprendre l'installation dans appdaemon, où se trouvent les fichiers, dans quels répertoires les copier, etc...

Je pense que tu as créé un outil au top, mais que la partie explication de l'installation est un peu light pour des hobbyistes comme moi.

Je suis sur HACF aussi, et je guette des explications complémentaires de ta part...

Cordialement.

Eric

**Comment by frtz13** [comment]
bonjour Eric,
les réponses à tes questions se trouvent dans le texte.
où tu en es?
appdaemon installé?
fichier appdaemon.yaml édité? (avec quoi? file éditor?)
addaemon démarre corretement? (connexion sur port 5050)
